dags/sustainment/information_url_checker.py

```diff
@@ -22,6 +22,10 @@
 CKAN = CKAN_CREDS[ACTIVE_ENV]["address"]
 CKAN_APIKEY = CKAN_CREDS[ACTIVE_ENV]["apikey"]
 
+# List of website we ignore for now,
+# since they are accessible but probably not allowed api calls
+white_list = ["www.ttc.ca", "www.ttc.ca/"]
```

Comment:
Do we need both of these? 
If not, let's get rid of it. We might have to add to this list in the future and it would be nice to keep it clean

Reply:
It currently has two formats,  https://www.ttc.ca/, http://www.ttc.ca. I can do a clean